package.json

```diff
@@ -36,11 +36,12 @@
     "@mixmaxhq/prettier-config": "^1.0.0",
     "@mixmaxhq/semantic-release-config": "^2.0.0",
     "cz-conventional-changelog": "^3.1.0",
-    "eslint": "^6.7.2",
-    "eslint-config-mixmax": "^4.2.0",
+    "eslint": "^6.8.0",
+    "eslint-config-mixmax": "^4.8.0",
     "express": "^4.15.2",
     "jasmine": "^2.5.3",
```

Comment:
can we get rid of jasmine now?

Reply:
oop ye 1842d4b702369a404e318de237b221a8eac29b94

Reply:
I was so confused as to why to get rid of `jasmine` because the `npm test` script still references it. Then I tried to run the script and nothing works. And then I realized the unit tests are of the `Jest` variety 😅